src/app/components/Document.jsx

```diff
@@ -1,7 +1,18 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import { AfterRoot, AfterData } from '@jaredpalmer/after';
 import { ServerStyleSheet } from 'styled-components';
 
+const resourceHints = () => (
```

Comment:
I had to pull these out into a separate function since CodeClimate didn't like having 26 lines in the render function.